copis/core.py

```diff
@@ -36,24 +27,30 @@
 from queue import Queue
 from typing import Any, List, Optional, Tuple
 
-import glm
 from pydispatch import dispatcher
 
 from enums import ActionType
-from gl.glutils import get_circle, get_helix
 from helpers import Point3, Point5
+from store import Store
+from coms import edsdk_object
+
+if sys.version_info.major < 3:
+    print("You need to run this on Python 3")
+    sys.exit(-1)
```

Comment:
I know there shouldn't be code before imports, but for the sake of this version check I think what it was originally was better (only have to import sys to check version, not everything)